app/models/container_volume.rb

```diff
@@ -0,0 +1,3 @@
+class ContainerVolume < ActiveRecord::Base
```

Comment:
I highly suggest using STI and having a subclass called ContainerVolumeKubernetes.  Later, when you want to tack on native creation, you'll have it, or when you need differences between that and, say, ContainerVolumneAmazon

This comment goes for all Container models.